galaxy_ng/app/api/v1/urls.py

```diff
@@ -0,0 +1,84 @@
+from django.urls import path
+
+from galaxy_ng.app.api.v1.views import LegacyRootView
+from galaxy_ng.app.api.v1.viewsets import LegacyRoleViewSet
+from galaxy_ng.app.api.v1.viewsets import LegacyUserViewSet
+
+
+urlpatterns = [
+    path(
+        'sync',
+        LegacyRoleViewSet.as_view({"post": "run_sync", "get": "get_task"}),
```

Comment:
I'm wondering if we should launch sync as a management command instead of an API call. Making this utility part of our public API seems unnecessary, since this should only be used for galaxy.ansible.com.

@awcrosby thoughts?

Reply:
It might be helpful to have this callable with a task status viewable since this may be called daily, or on demand, and able to be run by multiple users of the beta site.

Also we can delete this endpoint after this is adopted on galaxy.ansible.com

Reply:
The endpoint won't be available if the featureflag/env-var isn't set.